zeg/schemata/spec.yaml

```diff
@@ -13,6 +13,7 @@ properties:
     properties:
       path:
         type: string
+        pattern: \.(tab|tsv|csv|xlsx)$
```

Comment:
The extension shouldn't *actually* matter, is more a warning maybe?

Reply:
why only a warning? we dont want users creating broken collections

Reply:
The backend actually supports uploading a wider set of things, anything we can stuff into tabulator.